mail-client/scooby/scooby-0.9.9.recipe

```diff
@@ -0,0 +1,42 @@
+SUMMARY="Full featured e-mail client"
+DESCRIPTION="Scooby is an open source full featured BeOS native e-mail client."
+HOMEPAGE="https://sourceforge.net/p/scooby/"
+COPYRIGHT="Atsushi Takamatsu"
+LICENSE="GNU GPL v2"
+REVISION="1"
+srcGitRev="e53f65326329dd5a85fbdee640f4edefc2d043d3"
+SOURCE_URI="https://github.com/HaikuArchives/Scooby/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="29630d6ef1ff51cb463762d9bed23c8ebadac045e023c28b1ce1e8d72469384c"
+SOURCE_FILENAME="$portVersionedName.tar.gz"
+SOURCE_DIR="Scooby-$srcGitRev"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
```

Comment:
if you want to build for x86 on x86_gcc2 maybe add $secondaryArchSuffix at the needed places?

Reply:
@Begasus, we don't really need to add support for secondary architecture.

If it did not build on x86_gcc2 primary arch, or if it did provide at least one lib, or if the program would run much faster when built using a recent compiler, then, yes, it would be worth adding support for secondary architecture.

But maybe I'm missing something.

BTW, if you choose to keep support for secondary arch, then you need to add a SECONDARY_ARCHITECTURES="x86" as well as appropriate CONFLICTS (or small changes) to avoid overlaps between primary and secondary arch packages.

Reply:
Right there @fbrosson (ps nice to see you back!) :+1: 
I'm just wondering if x86 is still needed then?

Reply:
Thanks @Begasus ;-)
I guess we can keep x86 (i.e. avoid putting "?x86"), at least for the moment.

If a package builds fine on x86_gcc2 primary arch and on x86_64 then it will certainly also build fine on x86.
And in that case we can use
```
ARCHITECTURES="x86_gcc2 x86 x86_64"
```

For packages which don't build on x86_gcc2 but build fine for x86 secondary arch, we can probably avoid the "?" in front of the "x86" in ARCHITECTURES. We would have for such cases:
```
ARCHITECTURES="!x86_gcc2 x86 x86_64"
SECONDARY_ARCHITECTURES="x86"
```

I'd only use "?x86" (in ARCHITECTURES) in strange cases (if any) where a package would build on x86_64 but not on x86 (primary or secondary architectures). But I guess this should never happen.

Reply:
Sorry @nysnatuss, I forgot to tell that you may remove all occurences of `$secondaryArchSuffix` (and `$[secondaryArchSuffix}`).

Well, in case the crashes only happen with x86_gcc2 primary arch build (and not with x86 secondary arch builds on x86_gcc2 hybrid), then it might be worth keeping the secondaryArchSuffix, addng SECONDARY_ARCHITECTURES="x86" and changing "x86_gcc2" to "!x86_gcc2" in ARCHITECTURES.

Reply:
didn't check the $secondaryArchSuffix (yet) to see if the crash is also there ...

Reply:
FYI, with $secondaryArchSuffix it builds ok with x86_gcc2.

Reply:
if you still build for gcc2 yes, that deosn't effect the x86_gcc2 build (if you try to build for x86 (gcc5)) it fails ...
try "hp scooby_x86" ...

Reply:
first add SECONDARY_ARCHITECTURES="x86" as fbrosson already mentioned after the ARCHITECTURES to be able to trigger the x86 build

Reply:
We only add
```
SECONDARY_ARCHITECTURES="x86"
```
for packages which don't build on x86_gcc2 primary arch.
Here we are in the opposite situation. It only builds on x86_gcc2.
So what you can add is add:
```
SECONDARY_ARCHITECTURES="x86_gcc2"
```
It will allow users who build their own Haiku-x86 hybrid from source with support for x86_gcc2 secondary arch.
These users would then call
```
haikuporter scooby_x86_gcc2
```
after having added x86_gcc2 to the SECONDARY_ARCHITECTURES parameter in their haikuports.conf, of course.
See https://github.com/haikuports/haikuports/wiki/HaikuPorter-HaikuPortsConf for more details ;-)


Yes, this won't be very usefull to users of x86_64, unless we start shipping x86_64 hybrid (i.e. with support for x86_gcc2 as a secondary arch, like we did for x86 until a few months ago.)